### PR TITLE
Move location of marital status

### PR DESCRIPTION
**Pull request by akainic**
Moving marital status field out of spouse information section because not all users will now see this section, but marital status is required of all users.

Moving marital status to the demographic page after gender, and then making whether or not we show the spouse page conditional on if they are either Married or Separated.

### CODE REVIEW

File: src/client/routes.jsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const routes = [
       path="/household-information/financial-disclosure"/>,
   <Route
       component={SpouseInformationSection}
-      depends={{ understandsFinancialDisclosure: { value: 'Y' } }}
+      depends={{ understandsFinancialDisclosure: { value: 'Y' }, maritalStatus: { value: ['Married', 'Separated'] } }}
```
Comment (alexose) [inline src/client/routes.jsx]: The way I'd want to implement this is like so:

```
depends={[
  { understandsFinancialDisclosure: { value: 'Y' }, maritalStatus: { value: 'Married' }},
  { understandsFinancialDisclosure: { value: 'Y' }, maritalStatus: { value: 'Separated' }},
]}
```

**Comment by akainic** [comment]
New commit looks great, thanks @alexose! It would also be great if we could change the nav so the dependent sections don't show up there either.

**Review by markolson — APPROVED** [review]

**Comment by alexose** [comment]
Still getting a schema validation error on my end... trying to figure out why

**Comment by akainic** [comment]
@alexose wow! tests passed! does this look good to you? I'm going to create a separate ticket for fixing the nav.

**Comment by alexose** [comment]
👍  Will merge after CI

**Pull request merged**